01-main/packages/xpipe

```diff
@@ -0,0 +1,10 @@
+DEFVER=1
+ARCHS_SUPPORTED="x86_64 arm64"
+get_github_releases "xpipe-io/xpipe" "latest"
+if [ "${ACTION}" != "prettylist" ]; then
+    URL="$(grep "browser_download_url.*${HOST_ARCH}\.deb\"" "${CACHE_FILE}" | grep -v '.md5' | head -n1 | cut -d'"' -f4)"
```

Comment:
HOST_ARCH will be `amd64` or `arm64`
`X86_64` is HOST_CPU (which would give `aarch64` on arm) so we need a `case` or similar